plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -9,9 +9,9 @@ module Plutus.V1.Ledger.Api (
     -- * Validating scripts
     , validateScript
     -- * Cost model
-    , validateCostModelParams
-    , defaultBuiltinCostModelParams
-    , BuiltinCostModelParams
+    , validateCostModelData
+    , defaultCekCostModelData
+    , CostModelData
```

Comment:
It would be better not to change these unnecessarily if we can avoid it.

Reply:
I've changed Data back to Params, but `BultinCostModelParams` is juts wrong now.